### DISCUSSION TITLE
How to decrease the size of the APK?

### DISCUSSION THREAD

**Discussion by star-dust-one** [WalletConnect/walletconnect-monorepo#1796] [Developer Support]
The size of the APK with native SDK increased by 30MB. 
We use “Sign” and “Android-Core” libraries. Is it the expected size or it should be much smaller?

Example of the APK size with and without SDK:
<img width="1375" alt="Screenshot 2022-12-21 at 12 07 21" src="https://user-images.githubusercontent.com/106558245/208819638-b98dc0fc-1fdc-427e-8712-dadb5153de0a.png">

**Comment by Elyniss** [comment]
What two versions are you comparing? Is this an APK analyzer for your .apk or our SDKs?

**Reply by star-dust-one** [reply]
It's a comparison size of our APK with your SDK and without SDK. After including SDK in our project, the size of the app increases by 30 MB.

Library version:
```
"com.walletconnect:sign:2.3.1"
"com.walletconnect:android-core:1.5.0"
```
